editions/full/tiddlywiki.info

```diff
@@ -2,23 +2,18 @@
 	"description": "Full edition, containing all languages, themes and plugins",
 	"plugins": [
 		"tiddlywiki/bibtex",
-		"tiddlywiki/blog",
 		"tiddlywiki/browser-sniff",
 		"tiddlywiki/browser-storage",
-		"tiddlywiki/cecily",
 		"tiddlywiki/classictools",
 		"tiddlywiki/codemirror",
 		"tiddlywiki/comments",
-		"tiddlywiki/d3",
-		"tiddlywiki/github-fork-ribbon",
```

Comment:
I did update the github fork ribbon. It should work as expected again.

Reply:
I wonder whether marking the github fork ribbon plugin as "deprecated" is done by mistake. IMO it should have been marked "Legacy" or "Stable".

Reply:
Yes I think that's right, it should be marked "stable"